<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Dimesionamento</string>
     </attribute>
     <widget class="PlotWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>401</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>401</width>
        <height>181</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Bitola</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>As unit [mm²]</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>nº aprox.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>nº Barras</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>s [cm]</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>6.3</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>31.17</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>50.26</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>78.53</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>12.5</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>122.71</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>201.06</string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string>490.87</string>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>32</string>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string>804.24</string>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string>40</string>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string>1256.63</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>361</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Espaçamentos</string>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>17</y>
         <width>341</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Espaçamentos entre Barras</string>
       </property>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Horizontal [cm]</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Vertical [cm]</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>341</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>Espaçamento entre Estribos</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>s [cm]</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Entrada de Informações</string>
      </property>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>50</y>
         <width>191</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Seção </string>
       </property>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Base - bw [cm]</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>50</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cobrimento [cm]</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Altura - h [cm]</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>17</x>
          <y>80</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string> Alt. Útil - d [cm]</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Área de Aço Longitudinal [mm²]</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>203</x>
         <y>50</y>
         <width>151</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Material</string>
       </property>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>47</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Bitola</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>6.3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>47</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>φ agrg.</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1.9</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Resgatar Valores</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>361</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Estribos</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Bitola</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>5.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6.3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Vsw [N]</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Tramos</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>fyk</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>250</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>600</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
